.github/workflows/helm.yaml

```
@@ -37,7 +37,7 @@ jobs:
         uses: helm/kind-action@v1.2.0
         if: steps.list-changed.outputs.changed == 'true'
         with:
-          node_image: kindest/node:v1.22.2
+          node_image: kindest/node:v1.23.5
```

`helm/kind-action@v1.2.0` by default uses kind v0.11.1 (which allows us to use `kindest/node:v1.23.0`).
Also the action installs kubectl v1.20.8 which looks a bit older for k8s v1.23.

My suggestion is to specify all the versions of `kind`, `kindest/node`, and `kubectl`.

```suggestion
          version: v0.12.0
          node_image: kindest/node:v1.23.4
          kubectl_version: v1.23.4
```